server/wss.js

```diff
@@ -59,7 +59,11 @@ async function getStatusUpdateData(hospitalId) {
         ],
       },
     },
-  });
+  };
+  if (process.env.REACT_APP_PILOT_SHOW_ALL_RINGDOWNS === 'true') {
+    delete options.where.HospitalId;
+  }
```

Comment:
This removes the query filter so that all active ringdowns get sent to the ER view